sources/views/tabletoolbar.js

```diff
@@ -0,0 +1,94 @@
+import {JetView} from "webix-jet";
+import {icons} from "../models/icons";
+
+export default class TableToolbar extends JetView {
+	constructor(app, name, data) {
+		super(app, name);
+		this._gridData = data;
+	}
+
+	config() {
+		const _ = this.app.getService("locale")._;
+		const datatable = {
+			view: "datatable",
+			localId: "settings:datatable",
+			scroll: "y",
+			select: true,
+			editable: true,
+			columns: [
+				{
+					id: "value",
+					header: _("Value"),
+					editor: "text",
+					fillspace: 2
+				},
+				{},
+				{
+					view: "richselect",
+					id: "icon",
+					header: _("Icon"),
+					template: obj => `<span class="mdi mdi-${obj.Icon}"></span> ${obj.Icon}`,
```

Comment:
Template shows `undefined` for newly added elements (once you set a proper `Icon` id in the column). 
It would be great to show an empty string if there is no `obj.Icon`

Reply:
fixed